manual/intro.pro

```diff
@@ -21,7 +21,7 @@ and lets you add as much or as little metadata as you'd like.
 
 This introduction, in #prosidy, looks like this:
 
-#+example[syntax='prosidy']:
+#-example[syntax='prosidy']:
```

Comment:
Was this meant to change to `#=`?

Reply:
thank you for catching; fixing then pushing